src/Turbo/README.md

```diff
@@ -409,19 +410,23 @@ are passed to the template as variables: `entity`, `action` and `options`.
 
 ### Broadcast Conventions and Configuration
 
-The entity class **must** have a public property named `id` or a public method named `getId()`.
+When using the Mercure transport, the entity class **must** have a public property named `id` or a public method named `getId()`.
```

Comment:
This will most likely be mandatory for other transports too :)

Reply:
but they won't be forced to use the "id" property, nor to use property-access, etc.
They can implement their own id-retrieving/generation logic